pipeline/pipeline_runner.go

```diff
@@ -215,7 +215,7 @@ func Run(config *PipelineConfig) {
 			outputsWg.Done()
 			continue
 		}
-		LogInfo.Println("Output started: ", name)
+		LogInfo.Println("Output started:", name)
```

Comment:
Why the removal of the space? Looking at the other error messages a colon space seems to be more of the convention.